core/Cargo.toml

```diff
@@ -342,6 +343,11 @@ compio = { version = "0.11.0", optional = true, features = [
 ] }
 # for services-s3
 crc32c = { version = "0.6.6", optional = true }
+# for services-nebula-graph
+rust-nebula = { version = "0.1", optional = true, git = "https://github.com/nebula-contrib/rust-nebula", features = [
```

Comment:
Hi, this could prevent us from making a new release.

Reply:
I'll publish `rust-nebula` crate in a few days, you can take a look at my code first.